### ISSUE TITLE
Only apply correction algorithms / gain selection if not yet applied by EVB

### ISSUE THREAD

**Issue by maxnoe** [cta-observatory/ctapipe_io_lst#101]
Yesterday, the first data was taken with gain selection / deltaT correction applied by EVB.

We need smallish test data files for these conditions and need to implement.

**Comment by jsitarek** [comment]
Hi @maxnoe 
in addition, I think it would be good to be able to force switching on or off the individual corrections via a config switch. I'm pretty sure that we will have some data in which the tagging of which correction is used will not be done properly (which seems to be the case already now).

**Comment by maxnoe** [comment]
@jsitarek Yes, I was about to add that. There is currently no way of knowing which algorithms have already been applied without relying on external information like the run log or the shifters.

**Comment by maxnoe** [comment]
Implemented in #102.

I will leave this issue open, because we need to look at this again, once the EVB actually writes information about what algorithms have already been applied into the events.

**Comment by maxnoe** [comment]
Fixed and tested in #255
